config/webpack/webpack.config.js

```diff
@@ -70,6 +71,13 @@ module.exports = function (dirname, config = {}) {
             new CircularDependencyPlugin({
                 exclude: /(node_modules|examples)\/./,
                 failOnError: false // https://github.com/nodejs/readable-stream/issues/280#issuecomment-297076462
+            }),
+            new GlobalizePlugin({
+                production: false,
+                developmentLocale: "en",
+                supportedLocales: ["en", "es", "fr"],
+                messages: "../../src/i18n/[locale].json",
```

Comment:
There won't be `src` folder next week after this PR: https://github.com/theia-ide/theia/pull/299

We have to have a scalable solution for multiple extensions, also "../../src/i18n/[locale].json" is a relative path only for this repository. It is not going to work outside of it, then `theia-core` is consumed as a published node package.